nautobot/core/settings.py

```diff
@@ -293,6 +292,7 @@
     "taggit",
     "timezone_field",
     "nautobot.core",
+    "django.contrib.admin",
```

Comment:
Please add a comment here saying that this is must be below `nautobot.core` because templates.